webviz_ert/controllers/observation_response_controller.py

```diff
@@ -127,7 +127,8 @@ def update_graph(
 
         def _generate_plot(ensemble_id: str, color: str) -> ResponsePlotModel:
             ensemble = load_ensemble(parent, ensemble_id)
-            plot = _create_misfits_plot(ensemble.responses[response], [], color)
+            resp = ensemble.responses[str(response)]
```

Comment:
Is this refactoring really needed? Also, why is the string conversion required ? `str(response)`?

Reply:
Still, something is fishy here that you had to explicitly cast to string... Any way this function is a bit messy I think it should be refactored in the future such that it does not take `response` from the outer scope but it should expect a response name as part of the function arguments.